crates/db_schema/src/source/community.rs

```diff
@@ -137,5 +137,5 @@ pub struct CommunityFollower {
 pub struct CommunityFollowerForm {
   pub community_id: CommunityId,
   pub person_id: PersonId,
-  pub pending: bool,
+  pub pending: Option<bool>,
```

Comment:
It is never set to None, so i see no reason why it should be an option.

Reply:
Its a diesel thing we should be consistent about: a None value is possible, because the column has a default value. Designating that in the code helps us remember that the column has a default.

Reply:
Then we can remove the default value in this case?

Reply:
Nah its always a good idea to use a default value in SQL when the column is required, just in case.

Reply:
Hmm i really dont like to add things "just in case", because in the end no one will use it and its just useless bloat. If the column is required, there should be an error if its missing, and i dont see any reason to provide a default. It might even cause trouble if you forget to set the field, and get an unexpected default value instead of an error.

Reply:
In short:

Not null: Required
Null: Option<Option<...
**Not Null with default: Option** 
Null w default: Option<Option<...

Reply:
Yes i am suggesting to remove the default value for this column, and make the field required.